src/networks/data/local.json

```diff
@@ -1,7 +1,7 @@
 {
   "network": {
     "name": "local",
-    "chainId": "localtest",
+    "chainId": "localnet-1",
```

Comment:
I think this need to othter commit .. don't you @rootwarp?

Reply:
@Choi-Jinhong Do you prefer divide this into diff. issue?

Reply:
Nope. I knew we have to divide diff issue. But I don't care that this isn't mandatory :D

Reply:
I think it could be better apply this on another issue as your opinion.

Reply:
@Choi-Jinhong  I reverted default chainId. Please check again.